pkg/controller/orchestrator/orchestrator_reconcile.go

```diff
@@ -259,7 +259,7 @@ func (ou *orcUpdater) updateStatusFromOrc(insts InstancesSet, master *orc.Instan
 
 	// check if the master is up to date and is not downtime to remove in progress failover condition
 	if master != nil && master.SecondsSinceLastSeen.Valid && master.SecondsSinceLastSeen.Int64 < 5 {
-		log.Info("cluster failover finished", "master", master)
+		log.Info("cluster failover finished", "master", master.Key.Hostname)
```

Comment:
In my opinion, you can remove this log, it's not needed anymore.

Reply:
That would also fix #414 in that case. I'll push an update.